ios/RNRudderSdkModule.m

```diff
@@ -114,6 +114,36 @@ - (dispatch_queue_t)methodQueue
     [[RSClient sharedInstance] identify:_userId traits:_traits options:[self getRudderOptionsObject:_options]];
 }
 
+RCT_EXPORT_METHOD(alias:(NSString*)_newId options:(NSDictionary*)_options)
+{
+    if ([RSClient sharedInstance] == nil)
+    {
+        [RSLogger logWarn:@"Dropping the Alias call as RudderClient is not initialized yet, Please use `await` keyword with the setup call"];
+        return;
+    }
+    if([_newId isEqual:@""])
+    {
+        [[RSClient sharedInstance] alias:nil options:[self getRudderOptionsObject:_options]];
+        return;
+    }
+    [[RSClient sharedInstance] alias:_newId options:[self getRudderOptionsObject:_options]];
+}
+
+RCT_EXPORT_METHOD(group:(NSString*)_groupId traits:(NSDictionary*)_traits options:(NSDictionary*)_options)
+{
+    if ([RSClient sharedInstance] == nil)
+    {
+        [RSLogger logWarn:@"Dropping the Group call as RudderClient is not initialized yet, Please use `await` keyword with the setup call"];
+        return;
+    }
+    if([_groupId isEqual:@""])
+    {
+        [[RSClient sharedInstance] group:nil traits:_traits options:[self getRudderOptionsObject:_options]];
```

Comment:
We shouldn't support making group with `groupId` as nil/empty string, as it wouldn't be of much use of a group call with no groupId but group traits, as they are not persisted and lost in the subsequent events.

Maybe we can check if the `groupId` is an empty string on the React Native module itself and return if it's invalid.

We should check the same for Android as well.